### DISCUSSION TITLE
No network

### DISCUSSION THREAD

**Discussion by Fabianhoehne** [maca88/SmartBikeLights#75] [General]
Hey, im really struggling to set it up. 

I downloaded the app synced it to Garmin connect, created a new data field for it but it only shows "NoNetwork"  I tried it a while ago and gave up, because I couldn't figure out what I was doing wrong, today I tried again but it still was showing it.

Everything is connected to my network and synced with my phone .... I don't know what to do. 

I have a Garmin edge 1040 and a RTL 515 
![Uploading 20240618_040249.jpg…]()
![Uploading 20240618_040249.jpg…]()

**Comment by maca88** [comment]
Hi,

most likely you paired only the radar and not the light. Try to pair it again by using the approach from this video: https://www.youtube.com/watch?v=G-lgdH2mZwg

**Comment by Fabianhoehne** [comment]
Hey, thanks for the answer. Unfortunately I paired the light and everything.. a different add on which shows me the batterie life from the radar works just fine --Diese Nachricht wurde von meinem Android Mobiltelefon mit GMX Mail gesendet.Am 18.06.24, 21:59 schrieb maca88 ***@***.***>:

   
  Hi, 
  most likely you paired only the radar and not the light. Try to pair it again by using the approach from this video: https://www.youtube.com/watch?v=G-lgdH2mZwg

**Comment by maca88** [comment]
Can you provide the configuration that you are using? Did you set the serial number in the configuration? If the serial number does not match you will see "No network".

**Comment by Fabianhoehne** [comment]
Hey for sure, I'm glad You're helping me out 

###0,73535488:1,1303212406:1###6,5:Varia 515:0:16777215!2,:-1,Off:0!1,Solid:4!1,Peloton:5!1,Day Flash:7!1,Night Flash:6#0::#0:0#123!:123!#0#0#B3843##3#0#0

I tried to upload pictures to show you but I don't think that they are uploading correctly.  

Serial number is 3450696054

I
![Uploading 20240620_234653.jpg…]()

**Comment by maca88** [comment]
Try this configuration:

###0,73535488::1###6,5:Varia 515:0:16777215!2,:-1,Off:0!1,Solid:4!1,Peloton:5!1,Day Flash:7!1,Night Flash:6#0::#0:0#123!:123!#0#0#B3843##3#0#0

it is the same just without the serial number. If you have only one light then you don't need to set the serial number. The serial number is needed only when you have more than one taillight paired on you Garmin.

**Comment by Fabianhoehne** [comment]
I tried your configuration, and It still shows No network. I really don't know what I'm doing wrong ..

**Comment by maca88** [comment]
Hmm that is really odd. Could you try to reinstall the data field and leave the "Lights Configuration" app setting empty? If that doesn't work then I really have no cue what would be the issue. The data field should display the buttons or the light icons just by having the lights paired in the Garmin Sensors -> Lights menu

**Comment by corry98** [comment]
Hi maca thanks for the great app but I'm having the same issue where it just say no network, tried multiple settings with no luck, I'm using 1050 with RTL 515 only and it's not working for me

**Comment by corry98** [comment]
Never mind I found the issue guys, you need to pair radar and light separately(twice) in your Garmin device like maca said, then it would work
